packages/net/http/payload.pony

```diff
@@ -243,3 +243,10 @@ class iso Payload
     end
 
     list
+
+primitive Content
```

Comment:
The primitive can be alternatively called `MediaType` to match naming in [RFC7231][1] (emphasis added):
> The "Content-Type" header field indicates the **media type** of the associated representation <...>

[1]: http://tools.ietf.org/html/rfc7231#section-3.1.1.5